config/initializers/mailjet.rb

```diff
@@ -2,5 +2,8 @@
 Mailjet.configure do |config|
   config.api_key = ENV['MAILJET_API_KEY']
```

Comment:
check if env variables are defined before attributing them

Reply:
You are right with https://ruby-doc.org/core-1.9.3/ENV.html#method-c-fetch or `has_key?` and if not, skip Mailjet (like in `development` env) ?